frontend/src/styles/_message-list.scss

```
@@ -810,6 +810,73 @@
+  pointer-events: none;
+}
+
+@media (hover: none) {
+  .message-row-actions {
+    display: none;
+  }
+}
+
+.message-row-action {
+  @include icon-button(28px, 999px);
+  background: transparent;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)